views/partials/code-mirror-deps.ejs

```diff
@@ -0,0 +1,24 @@
+<!-- editor -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/codemirror.min.js"></script>
+<link type="text/css" rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/codemirror.min.css"
+/>
+<!-- json -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/mode/javascript/javascript.min.js"></script>
+
+<!-- linting -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/addon/lint/lint.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.39.0/addon/lint/json-lint.min.js"></script>
+<script src="//rawgithub.com/zaach/jsonlint/79b553fb65c192add9066da64043458981b3972b/lib/jsonlint.js"></script>
```

Comment:
Seems like there is more recent version of that library at https://github.com/circlecell/jsonlint-mod/blob/master/lib/jsonlint.js which has some extra fixes.